### PR TITLE
Update the ipfs config for ceramic-one

### PR DESCRIPTION
**Pull request by JustinaPetr**
@dbcfd it seems like the wheel still creates the config that defaults to bundled ipfs mode. My rust knowledge is quite limited but I took a stab at updating the config here so that we have the generated daemon config file that uses remote mode, has the ipfs host specified and has the peer data sync enabled. Please let me know if I am missing something.

### CODE REVIEW

File: ceramic-config/src/convert.rs [hunk 1]
```diff
@@ -69,19 +69,13 @@ impl Into<crate::daemon::DaemonConfig> for crate::Config {
             admin_dids: Some(self.http_api.admin_dids),
             cors_allowed_origins: cors,
         });
-        let ipfs = Some(if let crate::Ipfs::Remote(r) = self.ipfs {
-            crate::daemon::DaemonIpfsConfig {
+        let ipfs = Some(crate::daemon::DaemonIpfsConfig {
                 mode: Some(crate::daemon::DaemonIpfsConfigMode::Remote),
-                host: Some(r.host),
+                host: Some("http://localhost:5101".to_string()),
```
Comment (dbcfd) [inline ceramic-config/src/convert.rs]: So this will work, but doesn't let users change the value. We will want instead to change

https://github.com/ceramicstudio/wheel/blob/main/ceramic-config/src/lib.rs#L41
https://github.com/ceramicstudio/wheel/blob/main/ceramic-config/src/lib.rs#L19

You also might want to change this prompt https://github.com/ceramicstudio/wheel/blob/main/cli/src/prompt/ceramic_advanced_config.rs#L57

File: ceramic-config/src/daemon.rs [hunk 2]
```diff
@@ -18,6 +18,7 @@ pub struct DaemonIpfsConfig {
     pub host: Option<String>,
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub pinning_endpoints: Option<Vec<String>>,
+    pub disable_peer_data_sync: Option<bool>,
```
Comment (dbcfd) [inline ceramic-config/src/daemon.rs]: If we want to expose this to users will need to add it to https://github.com/ceramicstudio/wheel/blob/main/ceramic-config/src/lib.rs#L13 and expose it with prompts. We can do that in another PR if we want.

**Review by dbcfd — CHANGES_REQUESTED** [review]

**Comment by JustinaPetr** [comment]
@dbcfd Finally got the minute to get back at this. I made the suggested changes for the ipfs default implementation for `mode` and `ipfs`. Also updated the prompt. Would love to get your quick look here when you have a minute. If it looks good I will then open a new PR for the `disable_peer_data_sync`

**Review by dbcfd — APPROVED** [review]
